android/app/src/main/res/layout/search_item_speaker.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The speaker item from Session Details doesn't look good. Therefore i have used this